main/src/ca/uwaterloo/flix/language/phase/unification/set/SetFormula.scala

```diff
@@ -447,8 +420,8 @@ object SetFormula {
     case Compl(f1) => f1
     case inter@Inter(_, _, _, _, _, _, _) =>
       mkUnionAll(inter.mapSubformulas(mkCompl))
-    case union@Union(_, _, _, _, _, _, _) =>
-      mkInterAll(union.mapSubformulas(mkCompl))
+    case Union(l) =>
```

Comment:
is `l` unused? 

The pattern is inconsistent with the other cases. I like `union@Union(_)` better because you can read the case without having the figure out where f is bound

Reply:
Fixed